### PR TITLE
Feat/ot307 17 generate migration controller role

### PR DESCRIPTION
**Pull request by AlexMamani85**
# Ticket description: generate, migration and controller for Role

Ticket_number#17 (issue)

### CODE REVIEW

**Review by ryxtor — APPROVED** [review]
# Status: Approved 🟢
### Hello @AlexMamani85! 👋🏻
Well done with the Roles setup!

Your branch is complete! There is nothing else to say other than... it's time to merge it :shipit:

Congratulations! 🎉



Cheers and Happy coding!👏👏👏

File: app/models/role.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+#
+# Table name: roles
+#
+#  id          :bigint           not null, primary key
+#  description :string
+#  name        :string           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+class Role < ApplicationRecord
```
Comment (fciballero) [inline app/models/role.rb:13]: faltaría agregar una validación presence true del campo name

File: .gitignore [hunk 2]
```diff
@@ -11,7 +11,7 @@
 /tmp/
 
 # Used by dotenv library to load environment variables.
-# .env
+.env
```
Comment (fciballero) [inline .gitignore]: :+1:

**Review by fciballero — COMMENTED** [review]
Muy bien Alex, te dejo un par de comentarios.

**Pull request merged**